Resources/Prototypes/Actions/station_ai.yml

```
@@ -8,8 +8,12 @@
     priority: -9
     itemIconStyle: BigAction
     icon:
-      sprite: Interface/Actions/actions_ai.rsi
+#      sprite: Interface/Actions/actions_ai.rsi
+#      state: ai_core
+    ##SS220-resprite start
+      sprite: SS220/Interface/Actions/actions_ai.rsi
       state: ai_core
+    ##SS220-resprite end
```

Вот так везде сделать
```suggestion
      # SS220-resprite start
      #sprite: Interface/Actions/actions_ai.rsi
      sprite: SS220/Interface/Actions/actions_ai.rsi
      # SS220-resprite end
      state: ai_core
```